codedeps.yaml

```diff
@@ -1434,3 +1434,24 @@ data:
         //CMAKE_INSIST_FIND_PACKAGE_Einsums: true
         //CMAKE_DISABLE_FIND_PACKAGE_Einsums: true
 
+  - project: gauxc 
+    use:
+      added: 1.10
+      required: false
+      required_note: "Allow using the GauXC density functional library."
+      buildable: true
+      buildtime: true
+      cms: true
+    repository:
+      host: github
+      account: wavefunction91
+      name: GauXC
+      commit: master
```

Comment:
Is this how Psi4 pins to versions? Might want to be a bit careful here, always pinning to `master` might be a bit "living on the edge". I think this just motivates more that I need to push semantic versioning, I'll add that to my near-term TODO. For now, I would pick a commit that you know works and pin to it.

Reply:
ah, no, only the `use` and `conda` blocks are presently processed. `repository` and `cmake` are note taking to someday `sed` a couple lines in external/updatream/<project>/CMakeLists.txt and psi4/CMakeLists.txt (find_package() lines) for single-source-of-truch. and yes, we usually use tags or commits.

Reply:
Yeah, I made this as a "skeleton" fill-in for the codedeps file, especially since GauXC isn't on Conda. Would you be okay with me pinning to [a538336](https://github.com/wavefunction91/GauXC/pull/99/commits/a538336529c7fc58ac53e4b1947119ac4978573d)?

Reply:
Let's get all the QOL stuff checked in, then you can have a good commit to pin to

Reply:
Sounds good! Whatever PR we pin to, though, it should have https://github.com/wavefunction91/GauXC/pull/99 incorporated, because that PR allows the Psi4/GauXC Pytests to run successfully.

Reply:
https://github.com/wavefunction91/GauXC/pull/99 is merged, you can pin to current master if you'd like

Reply:
Fantastic! Done and done! I'll keep an eye out for other QoL updates that could be useful to us, but for now, that PR should be sufficient for us to pin to.